modules/rostests/apitests/atl/CImage.cpp

```diff
@@ -208,7 +208,7 @@ START_TEST(CImage)
     height = image2.GetHeight();
     ok(height == 48, "Expected height to be 48, was: %d\n", height);
     bpp = image2.GetBPP();
-    ok(bpp == 8, "Expected bpp to be 8, was: %d\n", bpp);
+    ok(bpp == 32, "Expected bpp to be 8, was: %d\n", bpp);
```

Comment:
can you please update the text indicating 8 as well? (same for the next 2)

Reply:
Done